pkg/cmd/util/term/terminal_writer.go

```diff
@@ -65,7 +65,17 @@ func (w wordWrapWriter) Write(p []byte) (nn int, err error) {
 	}
 	original := string(p)
 	wrapped := wordwrap.WrapString(original, w.limit)
-	return w.writer.Write([]byte(wrapped))
+	l, e := w.writer.Write([]byte(wrapped))
+
+	// This wrapped word writer could modify input in some cases such that
+	// the length of output is not equal to the length of input and this
+	// can cause ErrShortWrite error. So to avoid the error, make sure the
+	// returned length of output bytes is equal to the length of input.
+	// This returns length of original only in successful cases.
+	if e == nil {
+		l = len(original)
```

Comment:
actually, I see we're transforming the input here so len(p) != len(wrapped)... I'm not sure what to do in this case... returning a different number of bytes than we actually wrote can definitely break some composeability cases...